docs/install.rst

```diff
@@ -0,0 +1,352 @@
+.. highlight:: bash
+
+.. _install:
+
+==================
+Quick installation
+==================
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+Preface
+=======
+
+This part of the documentation covers the installation of CrateDB on Linux,
+macOS and Windows systems.
+The first step to using any software package is getting it properly installed.
+Please read this section carefully.
+
+
+Introduction
+============
+
+By installing CrateDB the way as outlined in this document, you will be able to
+quickly setup and run a single-node cluster.
+
+After the installation is finished, the CrateDB service should be up and
+running. Now, you can access the :ref:`Admin UI <admin-ui:index>` from your
+local machine by navigating to::
+
+    http://localhost:4200/
+
+After successfully installing the software, you might want to follow up with
+:ref:`taking the guided tour <use>`.
+
+Also, you might enjoy to be guided into further information about how to
+quickly adjust the :ref:`configuration settings <configure>`, read more
+details about the :ref:`reference:config` of CrateDB, the background about
+:ref:`howtos:bootstrap-checks`, multi-node configuration within the section
+about :ref:`howtos:clustering` and :ref:`howtos:going-into-production`. When
+operating a CrateDB cluster in production, `performance tuning`_ will also be
+of interest.
```

Comment:
Maybe it's just me, but I find this flow a bit weird. We point users to several steps before getting to the installation aspect. Users then have to remember this and come back after the installed it, or they get distracted with configuration before the started with the installation.

I think moving this to the end as a follow up would lead to a better flow.

Reply:
Thank you for your feedback. I did it like this on several occasions and had good experiences with it, but I will also be happy to put it at the end of the document. The main reason was that I didn't want to clutter each section for the individual installation target with such a paragraph.